Dockerfile

```diff
@@ -2,11 +2,7 @@
 # using the current working directory.
 #
 # This produces a docker image that contains a working Terraform binary along
-# with all of its source code, which is what gets released on hub.docker.com
-# as terraform:full. The main releases (terraform:latest, terraform:light and
-# the release tags) are lighter images including only the officially-released
-# binary from releases.hashicorp.com; these are built instead from
-# scripts/docker-release/Dockerfile-release.
+# with all of its source code.
```

Comment:
I do certainly agree about removing the specific reference to the file that no longer exists, but I worry that without any mention _at all_ of the other images it's likely that future readers will assume this file is what the official releases on Dockerhub are built from.

Perhaps, as a compromise:

```dockerfile
# This produces a docker image that contains a working Terraform binary along
# with all of its source code. This is not what produces the official releases
# in the "terraform" namespace on Dockerhub; those images include only
# the officially-released binary from releases.hashicorp.com and are
# built by the (closed-source) official release process.
```

Reply:
works for me! Updated.